Makefile

```diff
@@ -353,3 +353,30 @@ else
 		-r $(REPO_OWNER)/$(REPO_NAME) \
 		-Z > codecov-upload.log
 endif
+
+## -- Target for bumping up from a version 'n' to a version 'n+1'
+
+.PHONY :version-update
+## Upgrading operator version
+version-update:
```

Comment:
@Avni-Sharma what if we move those lines to a `hack` script?

Reply:
This should be set as guideline: if it is more than a check, then the logic should be moved to its own script.